mesh.yaml.template

```diff
@@ -12,7 +12,9 @@ cas:
 
 policy:
   namespace: $APP_NAMESPACE    # namespace on CAS instance `cas`
-  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed"
+  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed, aepic-leak"
+  ignore_advisories: "*"
```

Comment:
Do we need ignore_advisories?

Reply:
Yes, to ensure that we can run even on outdated servers